index.coffee

```diff
@@ -69,18 +12,25 @@ module.exports =
       default: ''
 
   activate: ->
-    prefix = 'linter-rubocop.'
     @subscriptions = new CompositeDisposable
-    @subscriptions.add atom.config.observe "#{prefix}executablePath",
-      (args) => @executablePath = if args then args.split ' ' else ['rubocop']
-    @subscriptions.add atom.config.observe "#{prefix}additionalArguments",
-      (args) => @additionalArguments = if args then args.split ' ' else []
+    @subscriptions.add atom.config.observe('linter-rubocop.executablePath', (executablePath) =>
+      @executablePath = executablePath
```

Comment:
This should be `executablePath.split(' ').filter((i) => i)` so we can support things like `rbenv @global blah blah rubocop` as commands.